src/sentry/workflow_engine/handlers/condition/slow_condition_query_handlers.py

```diff
@@ -85,21 +99,21 @@ def get_snuba_query_result(
 
     def get_chunked_result(
         self,
-        tsdb_function: Callable[..., Any],
         model: TSDBModel,
         group_ids: list[int],
         organization_id: int,
         start: datetime,
         end: datetime,
         environment_id: int | None,
         referrer_suffix: str,
-        conditions: list[tuple[str, str, str | None]] | None = None,
+        filters: list[dict[str, Any]] | None = None,
     ) -> dict[int, int]:
         batch_totals: dict[int, int] = defaultdict(int)
         group_id = group_ids[0]
+        conditions = self.get_extra_snuba_conditions(model, filters) if filters else []
```

Comment:
there's also some refactoring going on in this PR because i realized we don't need to generate the conditions until we get to here